### PR TITLE
Set up a "Hello world" project with two apps

### PR DESCRIPTION
**Pull request by NATASHA-ct**
### Project requirements
- Create a new React app called 'hello-react-front-end'
- Initialize your project with Git.
- Create the `App` component with react-router.
- Create the `Greeting` component that will display a greeting. Set it up as a route in your App component.
- Create a store, an action and a reducer that will connect to your API endpoint to get the random greeting.
- Display the random greeting in your `Greeting` component.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,225 @@
...
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (Nemwel-Boniface) [inline README.md]: - [x] Kindly note that in your README there are no clear instructions on how to setup the project to ensure that it displays random messages on the screen. When I visit the root route in your UI this is what I am seeing (Basically nothing)

image

- [x] Also, note that it is a requirement for you to include a link to the front backe end repo in the README of the front end and vice versa. You have instead given the link to the project download which is not part of the requirements. Kindly add this.

**Review by Nemwel-Boniface — CHANGES_REQUESTED** [review]
## Hello @NATASHA-ct     :wave: , 
I'm Nemwel your reviewer for this PR :man_beard: 

Good job so far! :clap: 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️  


## ♻️ Required Changes ♻️

- [x] Kindly note that it is a requirement for you to create pull requests in both the front-end and the back-end repos. Kindly ensure that you create a PR for your backend repo and update its README to include a link to the front-end repo as well as proper setup instructions.
- [x] Also, ensure that in the PR for the backend, you link the PR for the front end and vice versa.

_Check the comments under the review._

## [Optional] suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding! :partying_face: 👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Invalid Code Review Request ❌

Hi @NATASHA-ct, This is @sja-thedude your code reviewer.

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

You have submitted a project, that doesnt pass linters.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md [hunk 2]
```diff
@@ -1 +1,229 @@
...
+
+#### Link to backend repository
+    https://github.com/NATASHA-ct/Backend_Hello_Rails_React.git
+
+#### Link to backend repository pull request
+ 
+    https://github.com/NATASHA-ct/Backend_Hello_Rails_React/pull/1
+
+### Install
+
```
Comment (HAMMAS-SALEEM) [inline README.md:119]: - [x] In this section of readme at least these commands should exist which was requested by the previous reviewer 😊

**Frontend Readme** 💄
```javascript
### Install
git clone git@github.com:NATASHA-ct/Frontend_Hello_Rails_React.git
cd Frontend_Hello_Rails_React
npm install
npm start

```
**Backend Readme** 🔧
```ruby
### Install
git clone git@github.com:NATASHA-ct/Backend_Hello_Rails_React.git
cd Backend_Hello_Rails_React
bundle install install
rails s

```

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

## Hey @NATASHA-ct :wave:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following Gitflow
✔️ Following some requirements

Good job on implementing some of the requested changes, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by prolajumokeoni — APPROVED** [review]
Hi @NATASHA-ct 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- No linter errors ✅  
- Good design 💯 
- Good Pr title and description 💪🏿 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
- Please always follow proper gitflow in future projects 👍🏿 
- Please add the link to your backend repository to your frontend readme, this will make it easier to access your backend repository 👍🏿 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**